modules/test/playwright/tests/frontend-data-set-web/main/tests/advanced/filters.spec.ts

```diff
@@ -49,7 +49,9 @@ test(
 
 			await test.step('Check the results only show results with colors Blue, Green, and Yellow', async () => {
 				const blueCells = page.getByRole('cell', {name: 'Blue'});
-				const greenCells = page.getByRole('cell', {name: '🍏'});
+				const greenCells = page
+					.getByRole('cell', {name: 'Green'})
+					.or(page.getByRole('cell', {name: '🍏'}));
```

Comment:
To reduce code duplication, we can move all four `xCells` locators before steps, or in a Page.